### PR TITLE
Add instruction to install packages for Ubuntu

### PR DESCRIPTION
**Pull request by telezhnaya**

### CODE REVIEW

File: contrib/vscode/init.sh [hunk 1]
```diff
@@ -5,6 +5,15 @@ die () {
 	exit 1
 }
 
+SYSTEM_NAME="$(awk -F= '/^NAME/{print $2}' /etc/os-release)"
```
Comment (dscho) [inline contrib/vscode/init.sh]: That's a neat find! However, on other systems, that file may not even exist, and then the script shows a nasty error message that may leave the user puzzled. Also, we can avoid using `awk` altogether by doing this:

```sh
if grep '^NAME="Ubuntu"' /etc/os-release >/dev/null 2>&1
then
...
fi
```

Also, please note that Git's shell script conventions dictate the the `then` keyword lives on its own, separate line.

File: contrib/vscode/init.sh [hunk 2]
```diff
@@ -5,6 +5,15 @@ die () {
...
+if [ "$SYSTEM_NAME" = "\"Ubuntu\"" ] &&
+   [ ! -e /usr/include/openssl/ssl.h ]; then
```
Comment (dscho) [inline contrib/vscode/init.sh]: In Git's source code, we prefer `test` over `[`, and `-f` over `-e` for files.

Also, should we not test for more header files than just OpenSSL's? Like, `curl.h`, `expat.h` and `zlib.h`?

File: contrib/vscode/init.sh [hunk 3]
```diff
@@ -5,6 +5,15 @@ die () {
...
+    die "Some files missing. Try to run these commands in advance:
+    sudo apt-get update
+    sudo apt-get install git gcc make libssl-dev libcurl4-openssl-dev \\
+	    libexpat-dev tcl tk gettext git-email zlib1g-dev"
+fi
```
Comment (dscho) [inline contrib/vscode/init.sh]: Thank you. If you could add your `Signed-off-by:`, that would be awesome.

**Comment by telezhnaya** [comment]
Thank you for detailed review!

Please have a look on updated version.

I could add something like
`test ! -f /usr/include/x86_64-linux-gnu/curl/curl.h`
But I searched through this issue, I guess different users have different path to `curl.h`, and it's not only about 32 or 64 bits. So maybe it's better not to check for it at all.

**Comment by dscho** [comment]
As per private communication, I will close this PR.

**Pull request closed without merging**